docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -481,10 +481,17 @@ Methods
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
 | ``truncate``                | N                               |                                  |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``tz_convert``              | P                               | ``axis``, ``level``, ``copy``    |                                                    |
-+-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``tz_localize``             | P                               | ``axis``, ``level``, ``copy``,   |                                                    |
-|                             |                                 | ``ambiguous``, ``nonexistent``   |                                                    |
+| ``tz_convert``              | P                               | ``axis``, ``level``, ``copy``    | ``N`` if timezone format is not supported.         |
+|                             |                                 |                                  | Only timezones listed in ``pytz.all_timezones`` are|
+|                             |                                 |                                  | supported. So for example, ``UTC`` is supported but|
+|                             |                                 |                                  | ``UTC+/-<offset>`` such as ``UTC+09:00`` is not    |
+|                             |                                 |                                  | supported.                                         |
++-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
+| ``tz_localize``             | P                               | ``axis``, ``level``, ``copy``    | ``N`` if timezone format is not supported.         |
+|                             |                                 | ``ambiguous``, ``nonexistent``   | Only timezones listed in ``pytz.all_timezones`` are|
+|                             |                                 |                                  | supported. So for example, ``UTC`` is supported but|
+|                             |                                 |                                  | ``UTC+/-<offset>`` such as ``UTC+09:00`` is not    |
+|                             |                                 |                                  | supported.                                         |
```

Comment:
Nit:
"So" is unnecessary in "So for example", and we could also use the right punctuations to 
```
N if timezone format is not supported. Only timezones listed in pytz.all_timezones are supported. For example, UTC is supported, but UTC+/-<offset>, such as UTC+09:00 is not supported.
```

Reply:
Done.